week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,21 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
```

Comment:
What happens when you run this file? I think you'll find the answers don't match what you expect to get).
 (Hint: includes returns true or false. Does it make sense to compare this to a number?)